config/src/main/kotlin/org/springframework/security/config/web/server/ServerHttpSecurityDsl.kt

```diff
@@ -89,6 +90,33 @@ class ServerHttpSecurityDsl(private val http: ServerHttpSecurity, private val in
         this.http.securityMatcher(securityMatcher)
     }
 
+    /**
+     * Adds a [WebFilter] at a specific position.
```

Comment:
It would be nice to have an example of how to use this method, similar to what we have in `HttpSecurityDsl`.

The same applies for `addFilterBefore` and `addFilterAfter`.

Reply:
Done.